testing/test_capture.py

```diff
@@ -1492,3 +1492,8 @@ def test_fails():
     result_with_capture.stdout.fnmatch_lines(
         ["E * TypeError: write() argument must be str, not bytes"]
     )
+
+
+def test_stderr_write_returns_len():
```

Comment:
```suggestion
def test_stderr_write_returns_len(capsys):
```

This will ensure the test is testing what we expect even if the test suite is running with `-s`

Reply:
Great point, thanks! Glad to learn something new :)